providers/dns/dns_providers.go

```diff
@@ -79,6 +80,8 @@ func NewDNSChallengeProviderByName(name string) (acme.ChallengeProvider, error)
 		provider, err = ns1.NewDNSProvider()
 	case "otc":
 		provider, err = otc.NewDNSProvider()
+	case "lightsail":
+		provider, err = lightsail.NewDNSProvider()
```

Comment:
Kind of thinking it would be nice if this list was alphabetized now... it's getting long :smile: -- if you feel like helping out, would you alphabetize this list for us?